CMakeLists.txt

```diff
@@ -77,6 +77,10 @@ configure_file(
 
 if(MSVC)
     include("cmake/platform_win.cmake")
+
+    # CBL-6932 : Workaround to avoid crash when locking mutex on VC++ Runtime version < 14.4.
```

Comment:
Should this go before the above "include"?

Reply:
Why?